index.js

```diff
@@ -1 +1,265 @@
+let students_body = document.getElementById('students_data_body');
+let students_button = document.getElementById('students');
+let courses_body = document.getElementById('courses_data_body');
+let courses_button = document.getElementById('courses');
+let studentsURL = 'https://code-the-dream-school.github.io/JSONStudentsApp.github.io/Students.json';
+let coursesURL = 'https://code-the-dream-school.github.io/JSONStudentsApp.github.io/Courses.json';
+let studentsInfo = [];
+let coursesInfo = [];
+let submitNewStudent = document.getElementById('submit_btn');
+let editStudentButton = document.getElementById('edit-student-submit');
+
+/*==========================================
+ Event Listeners
+===========================================*/
+students_button.addEventListener('click', () =>{
+  showStudents();
+})
+
+courses_button.addEventListener('click', () =>{
+  showCourses();
+})
+submitNewStudent.addEventListener('click', (e) => {
+  addNewStudent();
+})
+
+
+/*============================================
+ Getting all data and storing in one function
+==============================================*/
+async function allData() {
```

Comment:
Since arrow functions are being mostly used in this code, it would be good to have some consistency. This function can be replaced by 
const allData = async () => {}
And it should work in the same way